config/locales/devise.he.yml

```diff
@@ -0,0 +1,61 @@
+---
+he:
+  devise:
+    confirmations:
+      confirmed: '&rlm;כתובת הדוא"ל אומתה בהצלחה.'
+      send_instructions: '&rlm;נשלח אליך דוא"ל עם הוראות לאימות כתובת הדוא"ל שאמור להתקבל בדקות הקרובות.'
+      send_paranoid_instructions: '&rlm;אם כתובת הדוא"ל שלך קיימת במסד הנתונים, יתקבל בדקות הקרובות דוא"ל עם הוראות לאימות כתובתך.'
+    failure:
+      already_authenticated: &rlm;חשבון זה כבר מחובר.
```

Comment:
When including "`&`" in YAML please surround it with single quotes like "`'&rlm;חשבון זה כבר מחובר.'`".